integration/common

```diff
@@ -34,166 +34,53 @@ waitForResponse () {
-  if curl -s $explorer_url >/dev/null; then
-    printf -- "[\033[32mrunning\033[0m]\n"
-    return
+  pushd tools/docker
+  # Grab logs from our node service and find the unlocked account
+  waitFor "docker logs chainlink-node 2>&1 | grep 'Unlocked account'" 10
+  export CHAINLINK_NODE_ADDRESS=`docker logs chainlink-node 2>&1 | grep 'Unlocked account' | awk '{print$5}'`
+
+  # If we're using geth for this test
+  # then we need to fund our hardcoded devnet address with the developer account given by geth
+  if [ $GETH_MODE ]
+  then
+    export DEVELOPER_ACCOUNT=`docker logs geth 2>&1 | grep "Using developer account" | awk -F "=" '{print$2}'`
+    yarn workspace @chainlink/integration-scripts fund-address
   fi
-  printf -- "[\033[31mstopped\033[0m]\n"
-  title "Starting explorer..."
-
-  yarn workspace @chainlink/json-api-client run build &>$SRCROOT/integration/logs/explorer-json-api-client.log
-  yarn workspace @chainlink/redux run build &>$SRCROOT/integration/logs/explorer-redux.log
-  yarn workspace @chainlink/explorer run build &>$SRCROOT/integration/logs/explorer-yarn.log
-  yarn workspace @chainlink/explorer run prod &>$SRCROOT/integration/logs/explorer.log & waitForResponse $explorer_url
-  title "Explorer is running."
-}
+  popd 
 
-add_clnode_to_explorer() {
-  title "Adding CL Node to Explorer..."
-
-  yarn workspace @chainlink/explorer admin:seed test-admin test-admin-password
-  clnode_name="test-$(date +%s)"
-  response=`EXPLORER_BASE_URL=$explorer_url EXPLORER_ADMIN_USERNAME=test-admin EXPLORER_ADMIN_PASSWORD=test-admin-password yarn workspace @chainlink/explorer run admin:clnodes:add $clnode_name`
-  export EXPLORER_ACCESS_KEY=$(echo $response | grep -o "AccessKey [a-zA-Z0-9]*" | awk '{print $2}')
-  export EXPLORER_SECRET=$(echo $response | grep -o "Secret [a-zA-Z0-9]*" | awk '{print $2}')
-  export EXPLORER_URL="ws://localhost:8080"
-  title "Added CL Node $clnode_name to Explorer."
-}
-
-setup_scripts() {
-  yarn --no-progress install &>$SRCROOT/integration/logs/yarn.log
-  yarn workspace chainlinkv0.5 build
-  yarn workspace chainlink build
-  yarn workspace @chainlink/integration-scripts setup
+  
 }
 
 deploy_contracts() {
   title "Migrating core contracts..."
   pushd integration >/dev/null
 
   # run migrations
-  yarn workspace @chainlink/integration-scripts deploy-contracts | tee $SRCROOT/integration/logs/deploy.log
-  export LINK_TOKEN_ADDRESS=`cat $SRCROOT/integration/logs/deploy.log | grep 'Deployed LinkToken at:' | awk '{print$4}'`
-  export ORACLE_CONTRACT_ADDRESS=`cat $SRCROOT/integration/logs/deploy.log | grep 'Deployed Oracle at:' | awk '{print$4}'`
-  export ETH_LOG_ADDRESS=`cat $SRCROOT/integration/logs/deploy.log | grep 'Deployed EthLog at:' | awk '{print$4}'`
-  export RUN_LOG_ADDRESS=`cat $SRCROOT/integration/logs/deploy.log | grep 'Deployed RunLog at:' | awk '{print$4}'`
+  yarn workspace @chainlink/integration-scripts deploy-contracts | tee /chainlink/integration/logs/deploy.log
```

Comment:
Hm, this is actually a subtree of the chainlink repository that is being called. So what do you think about `/usr/src/chainlink`? My initial thought was to make it a root level directory to keep our paths as short as possible. I know @felder-cl has aversion to this too, so I could change it if needed. I did put an open question in #devops about canonical paths for an app in docker, but it looks like the context of this is slightly different because we're not using any standard path at all here.

Reply:
I second the use of '/opt' here.  The '/usr/src' directory should be for source code, but this is more tooling in my opinion.  As for '/bin', '/etc', etc... I'm ambivalent.

Reply:
I've put the integration tests + source code into `/opt/chainlink` and the logs into `/var/log/chainlink/integration`. Does that satisfy the above requirements @j16r @felder-cl ?